conda/core/subdir_data.py

```diff
@@ -576,21 +512,55 @@ def maybe_decompress(filename, resp_content):
             """)
 
         else:
-            help_message = dals("""
-            An HTTP error occurred when trying to retrieve this URL.
-            HTTP errors are often intermittent, and a simple retry will get you on your way.
-            %s
-            """) % maybe_unquote(repr(e))
+            if url.startswith("https://repo.anaconda.com/"):
+                help_message = dals("""
+                An HTTP error occurred when trying to retrieve this URL.
+                HTTP errors are often intermittent, and a simple retry will get you on your way.
+
+                If your current network has https://www.anaconda.com blocked, please file
+                a support request with your network engineering team.
+
+                %s
+                """) % maybe_unquote(repr(e))
+            else:
+                help_message = dals("""
+                An HTTP error occurred when trying to retrieve this URL.
+                HTTP errors are often intermittent, and a simple retry will get you on your way.
+                %s
+                """) % maybe_unquote(repr(e))
 
         raise CondaHTTPError(help_message,
                              join_url(url, filename),
                              status_code,
                              getattr(e.response, 'reason', None),
                              getattr(e.response, 'elapsed', None),
-                             e.response)
-
-    except ValueError as e:
-        raise CondaIndexError("Invalid index file: {0}: {1}".format(join_url(url, filename), e))
```

Comment:
This `except ValueError` is no longer needed, because we're no longer doing `json.loads(raw_data)` in this function like we used to.